addons/udes_stock/models/stock_quant.py

```diff
@@ -64,11 +67,17 @@ def get_quantities_by_key(self, get_key=lambda q: q.product_id, only_available=F
             value = quant.quantity
             if only_available:
                 value -= quant.reserved_quantity
-            products[get_key(quant)] += value
+            if product_quantity is not None and product_quantity < value:
```

Comment:
I think it's safe to do `if product_quantity and product_quantity < value:` since if product_quantity is 0 then it will be false and if it is None it will be false as well.

Reply:
Removing after the call had with Miquel, will not added the optional parameter and will let like it was. Will handle on controller the change needed